test/addon_spec.rb

```diff
@@ -130,6 +225,86 @@
     end
   end
 
+  describe "#accept_eula" do
+    it "sets EULA as accepted" do
+      addon.accept_eula
+
+      expect(addon.eula_accepted).to eq(true)
+    end
+  end
+
+  describe "#refuse_eula" do
+    it "sets EULA as not accepted" do
+      addon.refuse_eula
+
+      expect(addon.eula_accepted).to eq(false)
+    end
+  end
+
+  describe "#eula_refused?" do
+    context "when EULA acceptance is not required" do
+      before do
+        allow(addon).to receive(:eula_acceptance_needed?).and_return(false)
+      end
+
+      it "returns false" do
+        expect(addon.eula_refused?).to eq(false)
+      end
+    end
+
+    context "when EULA acceptance is required" do
+      before do
+        allow(addon).to receive(:eula_acceptance_needed?).and_return(false)
+      end
+
+      context "and the license was accepted" do
+        it "returns false" do
+          addon.accept_eula
+
+          expect(addon.eula_refused?).to eq(false)
+        end
+      end
+
+      context "and the license was refused" do
+        it "returns true" do
```

Comment:
Ehm, does not seem so...

Reply:
:man_facepalming: You know, copy&paste.

Anyway, I have new changes that will remove this part.

Reply:
No problem, that's why we have the code review. :smiley: